### PR TITLE
Create MOM-2023-10-05.md

### PR DESCRIPTION
**Pull request by PedroDiez**
#### What type of PR is this?

Add one of the following kinds:
* documentation
* subproject management

#### What this PR does / why we need it:

Meeting Minutes, 5th October, 2023
Please review

### CODE REVIEW

File: documentation/MeetingMinutes/MOM-2023-10-05.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Required 'currency' information in API design - [Issue#12](https://github.com/camaraproject/BlockchainPublicAddress/issues/12) | Agreed to be closed. `CLOSED` after the meeting.
+API Enhancement due to Consent Management - [Issue#16](https://github.com/camaraproject/BlockchainPublicAddress/issues/16) | Pedro (TEF) illustrates the topic for anyone to have some background and also indicates have internally address this with their product team and their concern is that the checking for the permission for every transaction may be a stopper for business cases as it is not the best performance, also indicates that probably some product guidance from GSMA OGW would be helpful. Sebastian indicates that it should be avoided that personal information can be leaked by anyone and we should ensure the restricted access to this information. Georgios also comments not only the access to blockchain info but the knowledge of phonenumbers that can be used for spaming services. In this context, Celo comments about some work they have performed in [user identity in blockchain](https://docs.celo.org/protocol/identity/odis-use-case-phone-number-privacy).
```
Comment (sebKoeller) [inline documentation/MeetingMinutes/MOM-2023-10-05.md:36]:
```suggestion
API Enhancement due to Consent Management - [Issue#16](https://github.com/camaraproject/BlockchainPublicAddress/issues/16) | Pedro (TEF) illustrates the topic for anyone to have some background and also indicates have internally address this with their product team and their concern is that the checking for the permission for every transaction may be a stopper for business cases as it is not the best performance, also indicates that probably some product guidance from GSMA OGW would be helpful. Sebastian indicates that it should be avoided that personal information can be leaked by anyone and we should ensure the restricted access to this information. Georgios also comments not only the access to blockchain info but the knowledge of phone numbers that can be used for spaming services. In this context, Celo comments about some work they have performed in [user identity in blockchain](https://docs.celo.org/protocol/identity/odis-use-case-phone-number-privacy).
```

**Review by sebKoeller — COMMENTED** [review]
Found a small typo

File: documentation/MeetingMinutes/MOM-2023-10-05.md (hunk 1 above)
Comment (PedroDiez) [reply documentation/MeetingMinutes/MOM-2023-10-05.md:36]: thanks Seb!

**Review by sebKoeller — APPROVED** [review]
lgtm

**Comment by PedroDiez** [comment]
I will merge on 16/10

**Pull request merged**